sdk/programmableconnectivity/Azure.Communication.ProgrammableConnectivity/README.md

```diff
@@ -0,0 +1,47 @@
+# Azure ProgrammableConnectivity client library for .NET
+
+This SDK allows .NET applications to interact with Azure Programmable Connectivity Gateway, instead of using `curl` requests. For information on APC, see [docs](https://learn.microsoft.com/en-us/azure/programmable-connectivity/).
+
+  [Source code](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/programmableconnectivity/Azure.Communication.ProgrammableConnectivity/src) | [Package (NuGet)](https://www.nuget.org/packages/Azure.Communication.ProgrammableConnectivity) | [API reference documentation](https://azure.github.io/azure-sdk-for-net) | [Product documentation](https://docs.microsoft.com/azure)
+
+## Getting started
+### Install the package
+
+Install the client library for .NET with [NuGet](https://www.nuget.org/):
+
+```dotnetcli
+dotnet add package Azure.Communication.ProgrammableConnectivity --prerelease
+```
+
+### Prerequisites
+
+Before you can use the SDK successfully, you'll need to
+* Follow the [guide](https://learn.microsoft.com/en-us/azure/programmable-connectivity/azure-programmable-connectivity-create-gateway) to create a gateway, or have one already.
```

Comment:
```suggestion
* Follow the [guide](https://learn.microsoft.com/azure/programmable-connectivity/azure-programmable-connectivity-create-gateway) to create a gateway, or have one already.
```
nit:  You'll need to remove the language slugs from the MS Learn links.

Reply:
thank you for pointing out!